src/GameServer/RemoteView/World/AppearanceChangedPlugIn.cs

```diff
@@ -105,7 +105,7 @@ public async ValueTask AppearanceChangedAsync(Player changedPlayer, Item item)
         await connection.SendAppearanceChangedExtendedAsync(
             changedPlayer.GetId(this._player),
             item.ItemSlot,
-            (byte)(item.Definition?.Group ?? 0xFF),
+            (byte)((isEquipped? item.Definition?.Group : 0xFF) ?? 0xFF),
```

Comment:
If item was equipped, notify the actual item group byte value. Otherwise, mark as 0xFF so the client will know that it was unequipped